docs/source/conf.py

```diff
@@ -276,7 +276,7 @@
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
-man_pages = [(master_doc, "skada", "SKADA : Scikit Domain Adaptation", [author], 1)]
```

Comment:
It's my fault, I was not clear in my issue, but I want "Skada: Scikit Adaptation"